packages/xod-client/src/editor/components/CppImplementationEditor.jsx

```diff
@@ -40,7 +40,13 @@ const CppImplementationEditor = ({
         }
       },
       'Shift-Tab': CM => CM.indentSelection('subtract'),
+      'Ctrl-/': 'toggleComment',
+      'Cmd-/': 'toggleComment',
     },
+    showTrailingSpace: true,
+    autoCloseBrackets: true,
+    autoClearEmptyLines: true,
```

Comment:
I worry about this option. I find it harmful. Imagine a xoder just browses a project without intention to change anything. A simple cursor movement would easily *change* the project. Yes, it will be a simple whitespace removal, but it leads to save/discard question, false diffs for Git, etc…

I think it would be enough to just annoy a coder with the red dots and do not clear empty lines implicitly.

Reply:
Fixed, now it will clear empty lines only on "enter" keypress.
And clear only lines above the cursor, while it does not meet not empty line.